documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -94,7 +95,7 @@ func init() {
 
 func main() {
 	cfg := parseFlags()
-	http.Handle(cfg.telemetryPath, prometheus.Handler())
+	http.Handle(cfg.telemetryPath, promhttp.Handler())
```

Comment:
Well, it's an example, but this is a breaking change removing all the default instrumented metrics.